scripts/audio_processors/rtlsdr_noaa_apt_rx.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+##################################################
+# GNU Radio Python Flow Graph
+# Title: RTLSDR NOAA APT Receiver V1.0.0
+# Author: Dom Robinson
+# Description: APT to WAV recorder for Raspberry-Noaa -V2
+# Generated: Sun Apr  4 22:24:30 2021
+##################################################
+
+
+from gnuradio import analog
+from gnuradio import blocks
+from gnuradio import eng_notation
+from gnuradio import filter
+from gnuradio import gr
+from gnuradio.eng_option import eng_option
+from gnuradio.filter import firdes
+from optparse import OptionParser
+import osmosdr
+import time
+import sys
+
+class rtlsdr_noaa_apt_rx(gr.top_block):
+
+    def __init__(self):
+        gr.top_block.__init__(self, "RTLSDR NOAA APT Receiver V1.0.0")
+
+	##################################################
+	# Variables
+	##################################################
+
+    	# get some variables in place for inputs
+    	#
+    	# Arguments:
+    	#   1. Full path and name of stream file (including file extension)
+    	#   2. Gain to be used
+	#   3. Frequency (in Mhz)	
+    	#   4. Frequency offset (PPM)
+
+    	stream_name = sys.argv[1]
+	gain = float(sys.argv[2])
+	import decimal
+	freq = int(decimal.Decimal(sys.argv[3].strip("M"))*decimal.Decimal(1000000))
```

Comment:
This indentation is insane? lol

Reply:
Mostly autogenerated by GNURadio Composer (i take no blame :) )

Reply:
I don't understand how this isn't a syntax error lol

Reply:
My python fu isn't strong enough - not quite sure where the syntax error should be encountered?

Reply:
I read this deindent as important, I have no idea what's going on in reality lol

Reply:
That has now been corrected and i added some notes - marking as resolved.